README.rst

```diff
@@ -75,27 +75,101 @@ Awesome, isn’t it?
 Development
 ===========
 
-Run the tests
--------------
+You can follow the next steps in order to host postgraas_server locally and be able to develop features or bug fixes:
 
-You need to have docker installed
+Clone repository::
 
-Make sure you pull the right docker image::
+    git clone https://github.com/blue-yonder/postgraas_server
 
-    docker pull postgres:9.4
+*[optional]* Create and run a virtual environment
+
+*This step will help you to not install all needed python packages on your home computer, but in a virtual disposable environment so it doesn't cause any trouble with your local setup*::
 
-Make a virtualenv and install the requirements including the dev requirements and a local editable install
-of the package, for convenience you can install the requirements.in ::
+    python -m venv postgraas_server/
+    cd postgraas_server
+    source bin/activate
+
+Install all the project dependencies::
 
-    pip install -r requirements.in
     pip install -r requirements_dev.txt
+    pip install -r requirements_docker.in
+    pip install gunicorn
     pip install -e .
 
-For the tests you need a running postgres meta database and set some environment variables accordingly.
-There is a convenience script to set this all up using a docker postgres database::
+Docker
+-----------------
+
+Pull the right docker image::
+
+    docker pull postgres:9.4
+
+Your application.cfg file should look like this::
+
+    {
+        "metadb":
+        {
+            "db_name": "postgres",
+            "db_username": "postgres",
+            "db_pwd": "mysecret",
+            "host": "localhost",
+            "port": "5432"
+        },
+        "backend":
+        {
+            "type": "docker"
+        }
+    }
+
+Initialize a postgres DB within a docker container::
+
+    sh setup_integration_test_docker.sh
+
+Run a Docker container with the postgres image::
+
+    postgraas_init
+
+Postgres Cluster
+-----------------
+
+If you don't want to use Docker as the backend you could create a local postgres cluster
+
+Your application.cfg file should look like this::
```

Comment:
Should we just add `example_application.cfg`?

Reply:
You mean commit an example_application.cfg to the repository? I'm all for it

Reply:
Yeah, that was my basic idea.
Easier to reference that in the documentation than to explain what `example_application.cfg` should look like.